From 88ec924aba6e3301b97110994e7da18c2331faa7 Mon Sep 17 00:00:00 2001
From: Ben Pfaff <blp@nicira.com>
Date: Wed, 22 Sep 2010 09:52:27 -0700
Subject: [PATCH 1/6] xenserver: Remove LICENSE header that is difficult to keep up to date.

I had forgotten that I had added this header.  Let's keep all the
information about licensing in individual files instead.

Signed-off-by: Ben Pfaff <blp@nicira.com>
---
 xenserver/LICENSE |    5 -----
 1 files changed, 0 insertions(+), 5 deletions(-)

diff --git a/xenserver/LICENSE b/xenserver/LICENSE
index ce8949e..00fc4d8 100644
--- a/xenserver/LICENSE
+++ b/xenserver/LICENSE
@@ -1,8 +1,3 @@
-The files etc_xensource_scripts_vif and
-opt_xensource_libexec_interface-reconfigure are distributed under the
-terms of the GNU Lesser General Public License version 2.1 (included
-below).
-
 As a special exception to the GNU Lesser General Public License, you
 may link, statically or dynamically, a "work that uses the Library"
 with a publicly distributed version of the Library to produce an
-- 
1.7.1

